tests/output.bats

```diff
@@ -113,8 +158,8 @@ load '../lib/run'
 
   stub docker-compose \
     "-f docker-compose.yml -p buildkite1111 build --pull myservice : echo built myservice" \
-    "-f docker-compose.yml -p buildkite1111 up -d --scale myservice=0 : echo ran myservice dependencies" \
-    "-f docker-compose.yml -p buildkite1111 run --name buildkite1111_myservice_build_1 --rm myservice echo 'hello world' : echo ran myservice"
+    "-f docker-compose.yml -p buildkite1111 up -d --scale myservice=0 myservice : echo ran myservice dependencies" \
+    "-f docker-compose.yml -p buildkite1111 run --name buildkite1111_myservice_build_1 --rm myservice echo 'hello world' : echo ran myservice command"
 
   stub docker \
     "ps -a --filter label=com.docker.compose.project=buildkite1111 -q : echo" \
```

Comment:
in the next line there is also an issue with the stub:
```
bats-mock(docker): got docker ps -a --filter label=com.docker.compose.project=buildkite1111 --format {{.ID}}\t{{.Label "com.docker.compose.service"}}
bats-mock(docker): patterns  [3] = 'ps' '-a' '--filter' 
bats-mock(docker): arguments [6] = 'ps' '-a' '--filter' 'label=com.docker.compose.project=buildkite1111' '--format' '\{\{.ID\}\}\\t\{\{.Label\ \"com.docker.compose.service\"\}\}' 
bats-mock(docker): unexpected argument 'label=com.docker.compose.project=buildkite1111' at idx 3
bats-mock(docker): result 0
```

Reply:
Took me a while to figure out how to get this output; but I think I've managed to fix all the stub errors in the `output.bats` file.

It might be worthwhile adding the `BUILDKITE_AGENT_STUB_DEBUG=/dev/tty` variables commented out in the Docker Compose file for future ~adventurers~ developers 💖.